tsl/test/expected/compression_bgw.out

```diff
@@ -237,6 +237,7 @@ FROM conditions
 GROUP BY device, time_bucket(INTERVAL '1 hour', "time");
 NOTICE:  adding index _materialized_hypertable_8_device_day_idx ON _timescaledb_internal._materialized_hypertable_8 USING BTREE(device, day)
 REFRESH MATERIALIZED VIEW conditions_summary;
+WARNING:  REFRESH did not materialize the entire range since it was limited by the max_interval_per_job setting
```

Comment:
Why does this PR introduces this warning?

Reply:
Probably because the range was exceeded. The warning is harmless, but annoying. I'll see if I can get rid of it easily.

Reply:
This warning wasn't in the output before and I don't anything in the change, which would provoke it.

Reply:
I am also suspicious of this change.

Reply:
It disappeared when I removed the changes to `compress_utils`. I think this is a test error but will not fix it as part of this commit.